### PR TITLE
star: updating to 2.7.11a

### PR DESCRIPTION
**Pull request by A-N-Other**
Bumping this module file to include the latest release. Installed and tested on `arch=linux-rocky8-icelake`

### CODE REVIEW

**Review by alalazo — APPROVED** [review]

**Review by alalazo — CHANGES_REQUESTED** [review]
Pipelines are failing when building the new version of `star`

File: var/spack/repos/builtin/packages/star/package.py [hunk 1]
```diff
@@ -44,8 +45,12 @@ def edit(self, spec, prefix):
             env["CXXFLAGS_SIMD"] = ""
 
     def build(self, spec, prefix):
+        # different make targets if we're compiling for Mac M1/2
         with working_dir(self.build_directory):
-            make("STAR", "STARlong")
+            if spec.satisfies("target=aarch64:"):
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/star/package.py]: Given the name I wonder if:
```suggestion
            if spec.satisfies("platform=darwin target=aarch64: "):
```
is better. Which target do you have to make for `linux` on `aarch64`?

**Comment by A-N-Other** [comment]
Thanks @alalazo ... shall wait and see if it works.

**Comment by alalazo** [comment]
Apparently it didn't:
```
xxd -i parametersDefault > parametersDefault.xxd
/bin/sh: xxd: command not found
```

**Comment by A-N-Other** [comment]
Finally working on the ARM testbeds @alalazo. Should be good to go.

**Review by alalazo — APPROVED** [review]

**Pull request merged**